src/hooks/useCountries.jsx

```diff
@@ -1,16 +1,20 @@
 'use client'
 
 import { useQuery} from '@tanstack/react-query'
-import fetchData from './fetchData'
+import fetchData from '../app/api/fetchData'
 
 const useCountries = (options) => {
   return useQuery({
     queryKey: ['countries'],
```

Comment:
```suggestion
    getUseCountriesQueryKey(),
```

Po to właśnie ten util, żeby go użyć w tym miejscu i w `prefetchQuery`.